docs/getting-started/04-publish-adapter.md

```diff
@@ -1 +1,82 @@
-# Publishing your adapter
\ No newline at end of file
+# Publishing your adapter
+## one time to do for the release/publishing
+### ioBroker developer portal
+go to https://www.iobroker.dev/adapter-check
+
+Check your adapter by giving your adapter git repository. This will check a lot of important standarts. The following could be ignored:
+
+[E200] "Not found on npm. Please publish" 
+
+[W400] "Cannot find "your adapter" in latest repository
+
+because we have not published. Have a look to all other errors and warnings.
+
+### npm
+
+Log in to npmjs.com (best with your github account)
+
+Create access token for publish. These tokens may need to be regenerated and exchanged, when you ever has changed your account settings. Especialy when you changed your login method or security level(e.g. 2FA).
```

Comment:
We should add the note that the user needs to create "Automation tokens" (are they call that way?) to allow to ignore 2FA requirements

Reply:
Maybe that I did it already, but I can't remember. Is it that with npm login?

Reply:
It is after npm login when you create a npm token ... go there ... you need to choose a token type ... and you need to choose the "Automation token"